coordinator/src/main/scala/filodb.coordinator/queryplanner/MultiPartitionPlanner.scala

```diff
@@ -972,6 +973,20 @@ class MultiPartitionPlanner(val partitionLocationProvider: PartitionLocationProv
     PlanResult(Seq(resPlan))
   }
   //scalastyle:on method.length
+
+  def supportRemoteRawExport(lp: LogicalPlan): Boolean = {
+    val matchingColumnNameForDisabledTenants = queryConfig.routingConfig.stitchDisabledTenantColumn
+    val matchingColumnValues = lp.planColumnFilters().filter(_ match {
+      case ColumnFilter(name, Equals(_: String)) if name == matchingColumnNameForDisabledTenants         => true
+      case _                                                                                             => false
+    }).map(f => f.filter.valuesStrings.head.asInstanceOf[String]).toSet
+    // Check if any value in this set is present in disabled tenants
+    val isDisabledTenant = matchingColumnValues
+                            .intersect(queryConfig.routingConfig.tenantsWithDisabledRemoteStitch).nonEmpty
+    queryConfig.routingConfig.supportRemoteRawExport && !isDisabledTenant
```

Comment:
Could you use `isExportEnabledTenant`? (Just to get rid of that double-negative)